sonar/modules/users/jsonschemas/users/user-v1.0.0.json

```diff
@@ -2,46 +2,55 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "id": "https://sonar.ch/schemas/users/user-v1.0.0.json",
   "additionalProperties": false,
-  "title": "SONAR user v1.0.0",
+  "title": "User",
   "type": "object",
   "properties": {
     "$schema": {
       "type": "string",
       "default": "https://sonar.ch/schemas/users/user-v1.0.0.json"
     },
     "pid": {
-      "title": "User identifier",
-      "description": "User persistent identifier.",
-      "type": "string"
+      "title": "Identifier",
+      "type": "string",
+      "minLength": 1
     },
     "user_id": {
-      "title": "User account identifier",
-      "description": "User account identifier.",
-      "type": "integer"
+      "title": "Account identifier",
+      "type": "integer",
+      "minLength": 1
     },
     "full_name": {
       "title": "Name",
       "type": "string",
-      "minLength": 2
+      "minLength": 1
     },
     "birth_date": {
       "title": "Date of birth",
+      "description": "Date in yyyy-mm-dd format, ex: 1970-01-01",
       "type": "string",
+      "minLength": 1,
       "format": "date",
-      "pattern": "\\d{4}-((0[1-9])|(1[0-2]))-(((0[1-9])|[1-2][0-9])|(3[0-1]))$"
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}$"
     },
     "email": {
       "title": "Email",
       "type": "string",
+      "minLength": 1,
       "format": "email",
-      "pattern": "^.*@.*\\..*$",
-      "minLength": 6
+      "pattern": "^.+@.+\\..+$",
+      "form": {
+        "validation": {
+          "validators": {
+            "valueAlreadyExists": {}
+          }
+        }
+      }
```

Comment:
In RERO-ILS we used this pattern for email : `^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,})+$`
The minlength for an email is at least 6 characters : 1 before @, the @, 1 for domain name, 1 for dot, 2 for country

In RERO-iLS we also add a custom validation message if value doens't match the pattern : 
```
"validation": {
          "messages": {
            "pattern": "Email should have at least one <code>@</code> and one <code>.</code>"
          }
        }
```
It seems more human readable than just display a pattern

Reply:
I don't find the pattern `^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,})+$` in RERO-ILS. For the patrons the pattern seems to be `^.*@.*\\..*$`, which is similar to what I did.

Reply:
And `\w` does not match accented characters, is it a good choice?

Reply:
And email like this does not match: rero.sonar+superadmin@gmail.com..

Reply:
Finally, I will let the pattern `^.+@.+\\..+$`.